tweets/2022-01/opf-ds-meetup4.tweet

```diff
@@ -0,0 +1 @@
+Learn how to automate your data science workflow and run your notebooks in an automated fashion by creating an #AI pipeline using #Elyra, #Kubeflow, and #Tekton. Join us on Jan 25th, 11:00ET for Operate First Data Science Meetup. Mark your calendars now - https://t.co/BUJzlVesCY
```

Comment:
```suggestion
Learn how to automate your data science workflow and run your notebooks in an automated fashion by creating an #AI pipeline using #Elyra, #Kubeflow, and #Tekton. Join us on Jan 25th, 11:00ET for the Operate First Data Science Meetup. Mark your calendars now https://t.co/BUJzlVesCY
```

Reply:
Thanks for the suggestion but this is exceeding the word limit.

Reply:
I see. How about this?

Reply:
Looks good, thanks :)